src/offsetPath.js

```diff
@@ -2,6 +2,34 @@
 'use strict';
 
 (function () {
+  function basicShapePolygon (input) {
+    // TODO: Support the fill-rule option and %
+    var argumentList = input.split(',');
+    var coordinate = null;
+    var x = null;
+    var y = null;
+    var previousX = 0;
+    var previousY = 0;
+    var path = 'm 0 0';
```

Comment:
I suspect we should start with empty string and have m x y instead of l x y for the first point.